src/Uno.UI/UI/Xaml/Media/CompositionTarget.crossruntime.cs

```diff
@@ -1,33 +1,61 @@
 ﻿using System;
-
+using System.Runtime.CompilerServices;
+using System.Threading;
 using Uno.UI;
 using Uno.UI.Dispatching;
 
-namespace Microsoft.UI.Xaml.Media
+namespace Microsoft.UI.Xaml.Media;
+
+public partial class CompositionTarget
 {
-	public partial class CompositionTarget
+	private static Action _renderingActiveChanged;
+	private static bool _isRenderingActive;
+
+	public static event EventHandler<object> Rendering
 	{
-		public static event EventHandler<object> Rendering
+		add
 		{
-			add
-			{
-				NativeDispatcher.CheckThreadAccess();
+			NativeDispatcher.CheckThreadAccess();
 
-				var currentlyRaisingEvents = NativeDispatcher.Main.IsRendering;
-				NativeDispatcher.Main.Rendering += value;
-				NativeDispatcher.Main.RenderingEventThrottle = FeatureConfiguration.CompositionTarget.RenderEventThrottle;
-				NativeDispatcher.Main.RenderingEventArgsGenerator ??= (d => new RenderingEventArgs(d));
+			var currentlyRaisingEvents = NativeDispatcher.Main.IsRendering;
+			NativeDispatcher.Main.Rendering += value;
+			NativeDispatcher.Main.RenderingEventArgsGenerator ??= (d => new RenderingEventArgs(d));
 
-				if (!currentlyRaisingEvents)
-				{
-					NativeDispatcher.Main.WakeUp();
-				}
-			}
-			remove
+			IsRenderingActive = NativeDispatcher.Main.IsRendering;
+
+			if (!currentlyRaisingEvents)
 			{
-				NativeDispatcher.CheckThreadAccess();
+				NativeDispatcher.Main.WakeUp();
+			}
+		}
+		remove
+		{
+			NativeDispatcher.CheckThreadAccess();
+
+			NativeDispatcher.Main.Rendering -= value;
 
-				NativeDispatcher.Main.Rendering -= value;
+			IsRenderingActive = NativeDispatcher.Main.IsRendering;
+		}
+	}
+
+	internal static event Action RenderingActiveChanged
+	{
+		add => _renderingActiveChanged += value;
+		remove => _renderingActiveChanged -= value;
+	}
+
+	/// <summary>
+	/// Determines if the CompositionTarget rendering is active.
+	/// </summary>
+	internal static bool IsRenderingActive
```

Comment:
How does this new setup handle cases where an infinite animation is running and the application is backgrounded on mobile device? I guess the rendering should pause on backgrounding and resume with app resume?

Reply:
Apps in background are essentially pausing the process mobile, we don't need to do anything there.